cli/cli.go

```diff
@@ -138,6 +138,13 @@ func Main() {
 	deletePitrCmd.Flag("all", "Delete all chunks").Short('a').BoolVar(&deletePitr.all)
 	deletePitrCmd.Flag("force", "Force. Don't ask confirmation").Short('f').BoolVar(&deletePitr.force)
 
+	cleanupCmd := pbmCmd.Command("cleanup", "Delete Backups and PITR chunks")
+	cleanupOpts := cleanupOptions{}
+	cleanupCmd.Flag("older-than", fmt.Sprintf("Delete older than date/time in format %s or %s", datetimeFormat, dateFormat)).StringVar(&cleanupOpts.olderThan)
+	cleanupCmd.Flag("yes", "Don't ask confirmation").Short('y').BoolVar(&cleanupOpts.yes)
+	cleanupCmd.Flag("wait", "Wait for deletion done").Short('w').BoolVar(&cleanupOpts.wait)
+	cleanupCmd.Flag("wtimeout", "Wait timeout in seconds").Default("60").Uint32Var(&cleanupOpts.wtimeout)
```

Comment:
I don't think we need `wtimeout` flag. Can't see any use of it. @janwieremjewicz what do you think?

Reply:
Removed the flag. wait for 10 minutes. cannot wait endless but 10 minutes should be enough